infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -33,6 +34,10 @@ spec:
           env:
             - name: FEATURE_STORE_YAML_BASE64
               value: {{ .Values.feature_store_yaml_base64 }}
+            # - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            #   value: http://{{ .Values.service.name }}-collector.{{ .Release.namespace }}.svc.cluster.local:4317
+            # - name: OTEL_EXPORTER_OTLP_INSECURE
+            #   value: "true"   
```

Comment:
are we missing some logic here? should these be set if the values are set?

Reply:
Pls uncomment and use the configured value like `.Values.metrics.otelCollector.endpoint`, surrounded by the usual:
`{{- if .Values.metrics }}`